src/GitHub.VisualStudio.UI/Styles/ActionLinkButton.xaml

```diff
@@ -10,7 +10,7 @@
     <Setter Property="Template">
       <Setter.Value>
         <ControlTemplate TargetType="Button">
-          <TextBlock>
+          <TextBlock TextTrimming="CharacterEllipsis">
```

Comment:
Should I be aware of instances where we don't want this to be set? Is there a better way to approach this?